src/SFA.DAS.EAS.Web/Views/EmployerManageApprentices/Details.cshtml

```diff
@@ -70,37 +70,43 @@
         <h2 class="heading-medium visually-hidden">Apprentice status</h2>
         <table id="app-status" class="details-list details-list-new">
             <tbody>
+            <tr>
+                <th scope="row">Status</th>
+                <td>@Model.Data.Status</td>
+                <td id="editStatusLink">
+                    @if (Model.Data.CanEditStatus)
+                    {
+                        @Html.RouteLink("Edit status", "ChangeStatusSelectOption");
+                    }
+                </td>
+            </tr>
+            @if (Model.Data.PaymentStatus == PaymentStatus.Withdrawn)
+            {
                 <tr>
-                    <th scope="row">Status</th>
-                    <td>@Model.Data.Status</td>
-                    <td id="editStatusLink" class="link-right">
-                        @if (Model.Data.CanEditStatus)
+                    <th scope="row">Stop applies from</th>
+                    <td id="stopDate">
+                        @if (Model.Data.StopDate.HasValue)
                         {
-                            @Html.RouteLink("Edit status", "ChangeStatusSelectOption");
+                            @Model.Data.StopDate.Value.ToGdsFormat()
                         }
                     </td>
+                    
+                    @if (Model.Data.CanEditStopDate)
+                    {
+                        <td id="editStopDateLink" style="text-align:right">@Html.RouteLink("Edit", "EditStopDateOption")</td>
```

Comment:
Not sure on using the inline style here

Reply:
The removed code seemed to apply class="link-right"?

Reply:
Done